sdk/textanalytics/Azure.AI.TextAnalytics/tests/Infrastructure/TextAnalyticsClientLiveTestBase.cs

```diff
@@ -24,7 +24,11 @@ public TextAnalyticsClientLiveTestBase(bool isAsync, TextAnalyticsClientOptions.
             Sanitizer = new TextAnalyticsRecordedTestSanitizer();
         }
 
+        protected TextAnalyticsClient GetClient(AzureKeyCredential credential = default, TextAnalyticsClientOptions options = default, bool useTokenCredential = default)
+            => GetClient(out _, credential, options, useTokenCredential);
+
         public TextAnalyticsClient GetClient(
+            out TextAnalyticsClient nonInstrumentedClient,
```

Comment:
This is a really unusual pattern.   Can you help me understand the scenario behind this?

Reply:
We do the same in Form Recognizer. Example:

https://github.com/Azure/azure-sdk-for-net/blob/53c46a70140bcdc3041968fbbe0b1e30c282c1be/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisModels/OperationsLiveTests.cs#L34-L40

I'm not fully aware of how instrumentation works internally, but when we try to access the internal fields of an instrumented client like this: 
https://github.com/Azure/azure-sdk-for-net/blob/53c46a70140bcdc3041968fbbe0b1e30c282c1be/sdk/formrecognizer/Azure.AI.FormRecognizer/src/AnalyzeDocumentOperation.cs#L78-L79

they'll be null and we'll get a `NullReferenceException`. We only hit this code path in some live Operation tests.

There was a post in the .NET team channel related to that months ago, but I couldn't find it. Someone had run into the same problem in another library.

Reply:
Interesting.  Why the `out` variable rather than returning a tuple with both clients or having a dedicated method for creating an instrumented versus non-instrumented client?

Reply:
We didn't give it much thought, to tell the truth. Now that I think about it, having two separate methods could be better from a design perspective. Maybe `GetClient` and `GetNonInstrumentedClient`, to make it clear that the non-instrumented version is the odd one.

Reply:
I think the reasoning was that we were using both instrumented and non-instrumented clients in the same operation tests. We could just call `GetNonInstrumentedClient` + `InstrumentClient` it in the test body too.

Reply:
Don't feel obligated to change it on my account;  my intention was to understand not push on it.  😄